packages/illustrated-message/package.json

```diff
@@ -61,7 +61,7 @@
         "@spectrum-web-components/styles": "^0.23.3"
     },
     "devDependencies": {
-        "@spectrum-css/illustratedmessage": "^4.0.11"
+        "@spectrum-css/illustratedmessage": "^5.0.0-beta.0"
```

Comment:
Based on your notes and comms with design, I'd say we're good to make an official release of the CSS here and move this PR out of draft. Thanks for keeping this moving!